src/components/Infopic/Infopic.tsx

```diff
@@ -36,32 +37,28 @@ const TextComponent = ({
   buttonUrl,
 }: Omit<InfopicProps, "sectionIndex" | "image" | "alt">) => {
   return (
-    <div className="lg:max-w-lg">
-      <div className="mt-4">
-        <div>
-          <p className="text-subtitle pb-4 uppercase tracking-widest">
-            {subtitle}
-          </p>
-        </div>
-        <h1 className="text-secondary text-5xl font-semibold pb-4">{title}</h1>
-      </div>
+    <div className="flex flex-col gap-4 lg:px-8">
+      {subtitle && (
+        <p className="text-subtitle uppercase tracking-widest">{subtitle}</p>
+      )}
+      {title && (
+        <h1 className="text-secondary text-5xl font-semibold">{title}</h1>
+      )}
```

Comment:
question(blocking): sorry I missed out this part earlier. I think for infopic, minimally the `title` and the `imageUrl` should be required? Otherwise it is possible for the user to supply nothing and get nothing.

Reply:
yep agreed, for some reason I thought the props were purposely made optional by harish hahaha, will change 
in the spirit of making good components, I think it makes sense to make alt text compulsory too then?

[dc64df4](https://github.com/isomerpages/isomer-components/pull/5/commits/dc64df4d9a6ad9b1c4848c372e5b5df4b1b9e701)